apps/mobile/src/state/currentBtcPriceAtoms.ts

```diff
@@ -75,9 +75,8 @@ export const refreshBtcPriceActionAtom = atom(
 
     const fetchInfo = get(btcPriceDataAtom)[currency]
     if (
-      fetchInfo?.state === 'loading' ||
```

Comment:
@kaladivo I think `loading` check is not needed here as `fetchInfo.lastRefresh` should do the same thing